File: sample/src/main/java/ru/surfstudio/standard/interactor/common/error/NotModifiedException.java
package ru.surfstudio.standard.interactor.common.error;

import retrofit2.adapter.rxjava.HttpException;

/**
 * ошибка 304, см механизм Etag
 */
public class NotModifiedException extends HttpProtocolException {
    public NotModifiedException(HttpException cause, String message, int code, String url) {
        super(cause, message, code, url);
    }
}

File: template/src/main/java/ru/surfstudio/standard/interactor/common/error/NotModifiedException.kt
package ru.surfstudio.standard.interactor.common.error

import retrofit2.adapter.rxjava.HttpException

/**
 * ошибка 304, см механизм Etag
 */
class NotModifiedException(cause: HttpException, message: String, code: Int, url: String) : HttpProtocolException(cause, message, code, url)
